src/ape_plugins/_cli.py

```diff
@@ -194,7 +219,10 @@ def install(cli_ctx, skip_confirmation):
             #       installed packages, to potentially catastrophic results
             # NOTE: This is not abstracted into another function *on purpose*
 
-            args = [sys.executable, "-m", "pip", "install", "--quiet", package_name]
+            args = [sys.executable, "-m", "pip", "install", "--quiet"]
+            if upgrade_confirmation:
```

Comment:
I don't think this ever gets here though?
On line 212, we are checking if the plugin is not installed. If we are upgrading the plugin, then it will already be installed, which mean we will never get into this loop

Reply:
resolved with the latest push